modules/FormModel.js

```diff
@@ -113,86 +115,240 @@
     },
 
     /**
-     * Add a model that this form model should track against
-     * @method addModel
-     * @param modelConfig {Object} the Object model configuration you are tracking.
-     *   @param modelConfig.model {Backbone.Model} the object model
-     *   @param [modelConfig.fields] {Array} an array of strings where each String value corresponds to an attribute in the model.
-     *     Leave empty if you want to listen to all the fields.
-     * @param [copy=false] {Boolean} set to true if you want to make an initial pull from the object model upon adding.
+     * @method getMapping
+     * @param alias {String} the alias of the mapping - either a model mapping or a computed mapping
+     * @return the mapping config for that alias
      */
-    addModel: function(modelConfig, copy) {
-      this.__modelConfigs.push(modelConfig);
-      if (copy) {
-        this.__copyFields(modelConfig.fields, this, modelConfig.model);
-        this.__updateCache(modelConfig.model);
+    getMapping: function(alias) {
+      return this.__currentMappings[alias];
+    },
+
+    /**
+     * @method getMappings
+     * @return all the current mapping configs
+     */
+    getMappings: function() {
+      return this.__currentMappings;
+    },
+
+    /**
+     * Define or redefine how the form model pull/pushes or otherwise tracks properties between an object model(s).
+     * Examples:
+     * this.setMapping('modelAlias', true, optional model instance);
```

Comment:
Everytime I want to simplify the base cases I find that the fundamentals / structure of the mapping vs tracked models becomes cloudy or stretched. No-alias mappings don't make sense after they've been set. How would you change or remove this mapping but using the alias - you'd have to store the auto-generated value anyway. And adding a model.setTrackedModel(instance) that generates a mapping confuses the separation of mapping and tracked models. Currently to do what you are proposing model.setTrackedModel(instance) does would be:
model.setMapping('myModel', true, instance). In your opinion, is this bad enough to add a convenience method that blurs the separation of mapping and tracked models?